SpatialGDK/Documentation/known-issues.md

```diff
@@ -20,4 +20,5 @@ Known issue = any major user-facing bug or lack of user-facing feature that:
 | Seamless Travel is not supported | 2019-01-22 | [UNR-897](https://improbableio.atlassian.net/browse/UNR-897) | Disable Seamless travel |  
 | Opening the Editor while you are uploading or launching a cloud deployment disables the GDK Toolbar **Start** button, and Selecting **Stop** on the GDK Toolbar stops the running cloud deployment. | 2019-02-18 | [UNR-1006](https://improbableio.atlassian.net/browse/UNR-1006) | Wait for the cloud launch to finish, or launch the cloud deployment whilst the Editor is already open. |
 | When launching a client from a cloud deployment, it chooses the first **.exe** file in alphabetical order, rather than selecting a particular one. | 2019-03-06 | [LAUNCH-341](https://improbableio.atlassian.net/browse/LAUNCH-341) | When building your client, an underscore is added to the correct executable name. |
-| Replicated TimelineComponents are not supported. | 2019-03-14 | [UNR-1105](https://improbableio.atlassian.net/browse/UNR-1105) | Use an alternative method of achieving the desired behaviour, e.g. playing the timeline on a multicast event, or replicating what is changed by the timeline. |
+| NetMulticast RPCs do not work on singleton objects. | 2019-03-13 | [UNR-856](https://improbableio.atlassian.net/browse/UNR-856) | Do not use NetMulticast RPCs in Singletons.|
+| Replicated TimelineComponents are not supported. | 2019-03-14 | [UNR-1105](https://improbableio.atlassian.net/browse/UNR-1105) | Use an alternative method of achieving the desired behaviour, e.g. playing the timeline on a multicast event, or replicating what is changed by the timeline. |
```

Comment:
```suggestion
| Replicated TimelineComponents are not supported. | 2019-03-14 | [UNR-1105](https://improbableio.atlassian.net/browse/UNR-1105) | Use an alternative method of achieving the desired behaviour, such as playing the timeline on a multicast event, or replicating what is changed by the timeline. |
```
Just changing "e.g." because I know Laura would :D